src/test/java/com/meilisearch/integration/KeysTest.java

```diff
@@ -19,10 +21,6 @@ public class KeysTest extends AbstractIT {
     public void initialize() {
         this.setUp();
         this.setUpJacksonClient();
-    }
-
-    @AfterAll
```

Comment:
You should keep an afterall since the beforeEach is not triggered after the last tests. Resulting in the indexes created in the last test not being deleted